### ISSUE TITLE
Add Troublshooting section?

### ISSUE THREAD

**Issue by yarikoptic** [dandi/handbook#128]
which would likely just lead to our https://github.com/dandi/helpdesk/discussions .

one of the common cases could be outdated "system wide" Python which we no longer support, and then them following `pip install dandi` we instruct them to do in our UI

**Comment by kabilar** [comment]
I agree that we should add an explicit section to the left navigation.  How about we take the [Where to Get Help](https://www.dandiarchive.org/handbook/#where-to-get-help) section, simplify it, and use this content to create a new `User Support` section?

> one of the common cases could be outdated "system wide" Python which we no longer support, and then them following pip install dandi we instruct them to do in our UI

Sorry, not sure what you mean by the above statement in this context.

**Comment by yarikoptic** [comment]
on HPC, OSX etc system wide python might already be not supported "upstream". E.g. 3.7 is no longer supported by Python and by us.  But our download instructions on DLP say

![image](https://github.com/dandi/handbook/assets/39889/3be384f8-d059-4255-bbe4-a7030c117619)

NB update proposed in https://github.com/dandi/dandi-archive/pull/1919

Hence people would try to do that `pip install` and it might either fail or just install some older version of `dandi` which would not be working.

**Comment by kabilar** [comment]
Good catch, @yarikoptic.  In addition to your DLP pull request above, I have updated the sections in the docs here: #129.
